README.md

```diff
@@ -0,0 +1,158 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+</div>
+
+> # Blog App
+
+| Project Veiw Screenshots                                                                                              |
+| --------------------------------------------------------------------------------------------------------------------- |
+| <div align="center"><img src="./app/assets/images/recipe-app.png" alt="screenshot" width="auto" height="auto"/></div> |
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Video Presentation ](#video-project-presentation)
+- [💻 Getting Started](#getting-started)
```

Comment:
- [ ] It would be nice if you fix the Video Presentation link because it is broken 😨 Kindly fix it or remove it **(if you do not have any dedicated section for this link)** to make your readme descriptive and professional 😊 👍